probe_scraper/transform_probes.py

```diff
@@ -172,19 +170,18 @@ def transform(probe_data, node_data, break_by_channel):
            release channel.
     """
     channels = sorted_node_lists_by_channel(node_data)
-
-    result_data = {}
+    result_data =defaultdict(dict)
     for channel, channel_data in channels.iteritems():
-        print "\n" + channel + " - transforming probe data:"
+        print "\n" + str(channel) + " - transforming probe data:"
         for entry in channel_data:
             node_id = entry['node_id']
             readable_version = entry["version"]
             print "  from: " + str({"node": node_id, "version": readable_version})
             for probe_type, probes in probe_data[channel][node_id].iteritems():
                 # Group the probes by the release channel, if requested
+                result_data[probe_type][channel] = {"node": node_id, "version": readable_version} 
```

Comment:
You can create a new dictionary instead of reusing `result_data` and just use the new one for keeping track/counting the probes.

Reply:
OK. I will work on it.

Reply:
OK. Means I have Done Wrong this line.
**result_data[probe_type][channel] = {"node": node_id, "version": readable_version}**
You can Review it.

Reply:
@sannanansari , I would expect this line to become something like:

```
probe_counts[channel][probe_type] += 1
```